symbols.xml

```diff
@@ -3144,9 +3144,9 @@
         <global-address name='created_item_count'/>
```

Comment:
I don't know if I explained this well before, but you can delete the entries without addresses here as long as they're defined below.